packages/webapp/components/FooterNavBar.tsx

```diff
@@ -134,45 +159,61 @@ export default function FooterNavBar({
-                    tag="a"
-                    icon={tab.icon(index === selectedTab, unreadCount)}
-                    pressed={index === selectedTab}
-                    onClick={onItemClick[tab.path]}
-                  />
-                </LinkWithTooltip>
-              ) : (
-                <SimpleTooltip content={tab.title}>
-                  <Button
-                    {...buttonProps}
-                    icon={tab.icon(index === selectedTab, unreadCount)}
-                    onClick={() =>
-                      showLogin({ trigger: AuthTriggers.Bookmark })
+        {tabs.map(
+          (tab, index) =>
+            tab.component ||
+            (tab.requiresLogin && !user ? null : (
+              <div key={tab.title} className="relative">
+                {!tab.shouldShowLogin || user ? (
+                  <LinkWithTooltip
+                    href={
+                      typeof tab.path === 'string' ? tab.path : tab.path(user)
                     }
-                  />
-                </SimpleTooltip>
-              )}
-              <Flipped flipId="activeTabIndicator">
-                {selectedTab === index && (
-                  <ActiveTabIndicator
-                    className="w-12"
-                    style={{ top: '-0.125rem' }}
-                  />
+                    prefetch={false}
+                    passHref
+                    tooltip={{ content: tab.title }}
+                  >
+                    <Button
+                      {...buttonProps}
+                      tag="a"
+                      icon={tab.icon(index === selectedTab, unreadCount)}
+                      iconPosition={ButtonIconPosition.Top}
+                      pressed={index === selectedTab}
+                      onClick={
+                        onItemClick[
+                          typeof tab.path === 'string'
+                            ? tab.path
+                            : tab.path(user)
+                        ]
+                      }
+                    >
+                      {tab.title}
+                    </Button>
+                  </LinkWithTooltip>
+                ) : (
+                  <SimpleTooltip content={tab.title}>
+                    <Button
+                      {...buttonProps}
+                      icon={tab.icon(index === selectedTab, unreadCount)}
+                      iconPosition={ButtonIconPosition.Top}
+                      onClick={() =>
+                        showLogin({ trigger: AuthTriggers.Bookmark })
+                      }
+                    >
+                      {tab.title}
+                    </Button>
+                  </SimpleTooltip>
                 )}
-              </Flipped>
-            </div>
-          ),
+                <Flipped flipId="activeTabIndicator">
+                  {selectedTab === index && (
+                    <ActiveTabIndicator
+                      className="w-12"
+                      style={{ top: '-0.125rem' }}
```

Comment:
Let's please use tailwind here 🙏

Reply:
I could not find a tailwind class to apply that fits this style, do you happen to know if there is one?

Reply:
`-top-0.5` should do